gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyRunner.java

```diff
@@ -52,7 +53,8 @@ public DichotomyResult<RaoResponse> run(SweData sweData, DichotomyDirection dire
         Parameters parameters = dichotomyConfiguration.getParameters().get(direction);
         dichotomyLogging.logStartDichotomy(direction, parameters);
         DichotomyEngine<RaoResponse> engine = buildDichotomyEngine(sweData, direction, parameters);
-        return engine.run(sweData.getNetwork());
+        Network network = NetworkShifterProvider.getNetworkByDirection(sweData, direction);
```

Comment:
why would you make this method static ? Did you actually or was it someone else ?